extension/src/Runner/runnerResultStream.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as events from "events";
+import { Readable } from "stream";
+
+export class RunnerResultStream extends events.EventEmitter implements IRunnerResultStreamEvent {
+    constructor(public readonly stderr: Readable, public readonly stdout: Readable) {
+        super();
+    }
+}
+
+interface IRunnerResultStreamEvent {
+    on(event: 'finish', listener: () => void): this;

```

Comment:
'this' as return type. What does it mean?

Reply:
it follows `events.EventEmitter`. However, maybe I should rename the interface to IRunnerResultStreamEventEmitter